app-editors/vim/vim-8.0.1230.recipe

```diff
@@ -134,11 +134,10 @@ INSTALL()
 {
 	make install
 
-	# Make arrow keys, backspace, delete, etc behave as expected.
-	echo "set nocompatible" > $dataDir/vim/vimrc
```

Comment:
What happend with this?

Reply:
@extrowerk - See @alaviss for recent changes. Latest recipe still builds successfully.

Reply:
He haven't said you should remove this.

Reply:
I did, read the diff https://github.com/haikuports/haikuports/pull/3092#discussion_r219049054 . An explanation is provided also.

Reply:
Ah, true, my bad. (Actually Firefox is guilty here, as it cannot open the outdated comments, old version, cannot update.)

Reply:
Implemented.